.evergreen/install-dependencies.sh

```diff
@@ -41,14 +41,16 @@ path: $NVM_SYMLINK
 EOT
 
   nvm install ${NODE_VERSION}
+  nvm install 10.22.0 # install lts for compilation only
```

Comment:
should we use `nvm install --lts=dubnium` instead?

Reply:
Windows nvm doesn't appear to support an lts alias, I can make a ticket to add some curl work to get the latest dubnium.

Reply:
oh duh, right right sorry I forgot about nvm-windows